Fix publishing issues

```
> Task :accelerometer:publishMavenPublicationToSonatypeRepository FAILED
FAILURE: Build failed with an exception.


> Failed to publish publication 'maven' to repository 'sonatype'
   > Could not PUT 'https://oss.sonatype.org/content/repositories/snapshots/com/gluonhq/attach/accelerometer/4.0.23-SNAPSHOT/accelerometer-4.0.23-20250703.083022-1.jar'. Received status code 403 from server: Forbidden
```

--- a/build.gradle
+++ b/build.gradle
@@ -41,6 +41,8 @@ nexusPublishing {
     packageGroup = 'com.gluonhq'
     repositories {
         sonatype {
+            nexusUrl.set(uri("https://ossrh-staging-api.central.sonatype.com/service/local/"))
+            snapshotRepositoryUrl.set(uri("https://central.sonatype.com/repository/maven-snapshots/"))
             username = project.hasProperty('sonatypeUsername') ? project.property('sonatypeUsername') : ''
             password = project.hasProperty('sonatypePassword') ? project.property('sonatypePassword') : ''
         }

